lib/ex_webrtc/peer_connection.ex

```diff
@@ -619,19 +623,71 @@ defmodule ExWebRTC.PeerConnection do
     end
   end
 
-  defp update_transceivers(%{transceivers: transceivers, config: config}, sdp) do
-    Enum.reduce_while(sdp.media, {:ok, transceivers}, fn mline, {:ok, transceivers} ->
-      case ExSDP.Media.get_attribute(mline, :mid) do
-        {:mid, mid} ->
-          transceivers = RTPTransceiver.update_or_create(transceivers, mid, mline, config)
-          {:cont, {:ok, transceivers}}
+  defp update_transceivers(state, sdp) do
+    Enum.reduce(sdp.media, state.transceivers, fn mline, transceivers ->
+      {:mid, mid} = ExSDP.Media.get_attribute(mline, :mid)
+
+      # TODO: reusing transceivers
```

Comment:
Could we elaborate on this comment? I'm slowly getting lost

Reply:
I meant "recycling transceivers", we don't do that at the moment.